src/main/java/success/planfit/domain/user/User.java

```diff
@@ -0,0 +1,52 @@
+package success.planfit.domain.user;
+
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+
+@Getter
+@NoArgsConstructor
+@Entity
+@DiscriminatorColumn(name="authorized_by", discriminatorType = DiscriminatorType.STRING)
+public class User {
```

Comment:
```suggestion
public abstract class User {
```

저희 로직에서는 `User` 인스턴스를 직접 만들 상황은 없을 것 같고, 모든 상황에서 `User` 객체를 상속하는 `GoogleUser`, `KakaoUser` 등의 인스턴스만 생성할 것 같습니다.
그렇다면 `User` 인스턴스를 생성하지 못하도록 추상 클래스로 만드는 것은 어떻게 생각하시나요?

Reply:
넵 좋습니다! 수정하겠습니다